.github/workflows/tox.yml

```diff
@@ -1,9 +1,8 @@
 on:
   push:
     branches:
-      - master
-  pull_request:
-
+      - "**"
+  workflow_dispatch:
```

Comment:
** provides execution for all branch pushes and workflow_dispatch provides manuel execution